### PR TITLE
[PAGOPA-1099] New workflows + JavaAgent

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

🟢 increase heap memory and pod memory
and...
🗑️ deleted `.devops` folder
➕ new GHA workflow
🐳 edit Dockerfile with openTelemetry agent
🔁 replaced google style with intellij code style
🆙 improved some scripts (es: generate openapi with junit test)
✏️ updated readme and `.editorconfig`

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: helm/values-prod.yaml [hunk 1]
```diff
@@ -99,6 +99,12 @@ basecache: &basecache
     CRON_JOB_ICA_SCHEDULE_ENABLED: "false"
     CRON_JOB_ICA_SCHEDULE_EXPRESSION: "0 0 0 * * *"
     AZURE_STORAGE_TABLE_NAME: "pagopapweuapiconfigsaicatable"
+    JAVA_OPTS: "-XX:+PrintFlagsFinal -XX:MaxHeapSize=2g"
+    OTEL_SERVICE_NAME: "pagopa-api-config"
+    OTEL_RESOURCE_ATTRIBUTES: "deployment.environment=azure-dev"
```
Comment (jacopocarlini) [inline helm/values-prod.yaml:104]:
```suggestion
    OTEL_RESOURCE_ATTRIBUTES: "deployment.environment=azure-prod"
```

**Review by andrea-deri — COMMENTED** [review]
Good job! 🔝

**Review by andrea-deri — APPROVED** [review]

File: openapi/generate_openapi_and_swagger.sh [hunk 2]
```diff
@@ -9,35 +9,15 @@ if [[ "$(pwd)" =~ .*"openapi".* ]]; then
...
-# waiting the service
-printf 'Waiting for the service'
-attempt_counter=0
-max_attempts=50
-until $(curl --output /dev/null --silent --head --fail http://localhost:8080/apiconfig/api/v1/info); do
-    if [ ${attempt_counter} -eq ${max_attempts} ];then
-      echo "Max attempts reached"
-      exit 1
-    fi
+mvn test -Dtest=OpenApiGenerationTest
```
Comment (pasqualespica) [inline openapi/generate_openapi_and_swagger.sh:12]: great solution 💯

**Review by pasqualespica — APPROVED** [review]
great work update [census page](https://pagopa.atlassian.net/wiki/spaces/IQCGJ/pages/736002132/Census+Repo+Status) accordingly too

**Pull request merged**